### ISSUE TITLE
withAuthenticator component not centered

### ISSUE THREAD

**Issue by MartijnGevaert** [aws-amplify/amplify-ui#1756]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Authenticator

### How is your app built?

Create React App

### Please describe your bug.

For version "@aws-amplify/ui-react": "~2.11.0", the withAuthenticator sign in component was centered correctly.

After that version (2.12.0 and onward), the withAuthenticator component is no longer centered.

### What's the expected behaviour?

The expected behavior is for the authenticator to be centered.

### Help us reproduce the bug!

I believe simply using the withAuthenticator higher order component should show the behavior.

### Code Snippet

```typescript
// I think I would need to set up a whole new amplify project to include a minimal example for this (not sure though)
```

### Additional information and screenshots

Version 2.11.0
![image](https://user-images.githubusercontent.com/41551215/164567912-3f959a5b-1445-405a-9eda-f5c8b9152b24.png)

Version 2.12.0 and onward
![image](https://user-images.githubusercontent.com/41551215/164567639-f66aa7dc-4765-4154-9571-0d7bdd933f26.png)

**Comment by Ryuuji159** [comment]
Any news about this? I'm having the same problem

**Comment by calebpollman** [comment]
Thanks for raising this issue @MartijnGevaert. Was able to reproduce the issue in the `default` variant of the `Authenticator`.  We should have a fix in our next release

**Comment by wlee221** [comment]
The fix has been released to `@aws-amplify/ui-react@2.15.6`. Please feel free to reopen this issue if issue persists, thanks!

**Comment by MartijnGevaert** [comment]
I can confirm this new release fixes it. Thanks for the quick fix.
